conda/recipe/0001-c++14.patch

```diff
@@ -0,0 +1,25 @@
+From 286b45047a4bb7036896d27c1201955ed3692593 Mon Sep 17 00:00:00 2001
+From: "Andres A. Lugo-Reyes" <aalugore@us.ibm.com>
+Date: Mon, 22 Jul 2019 21:02:50 +0000
+Subject: [PATCH] set std=c++14
+
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 7e07808..31f6206 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -97,7 +97,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
+ set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++14")
+ 
+ # CXX flags
+-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-unused-variable -Wno-unused-function -fno-strict-aliasing -fPIC -fvisibility=hidden")
++set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wno-unused-variable -Wno-unused-function -fno-strict-aliasing -fPIC -fvisibility=hidden")
+ 
+ if (WERROR)
```

Comment:
Can you rebase to the latest DALI version. Then the above and this `WERROR` should not be needed.

Reply:
This line isn't actually being added by the patch. Only lines 18 and 19 actually affect the code. The `WERROR` in question is still present in master https://github.com/NVIDIA/DALI/blob/master/CMakeLists.txt#L102. In any case, this patch doesnt touch that line. I rebased just in case.